.github/workflows/helm.yml

```diff
@@ -0,0 +1,36 @@
+name: 'Helm release'
+
+on:
+  push:
+    paths:
+      - 'helm-charts/l7mp-operator/Chart.yml'
+      - 'helm-charts/l7mp-ingress/Chart.yml'
+
```

Comment:
can it have an additional condition similar to the folowing?
```
on:
  push:
    branches: [ master ]
```

Reply:
Please see the changes in [40ee500](https://github.com/l7mp/l7mp/pull/99/commits/40ee500edcddf51e984fbea55d0cd4c255bbe4ba).